### DISCUSSION TITLE
Unable to upload certificate of enrollment for Student Developer Pack application

### DISCUSSION THREAD

**Discussion by Githarold** [community/community#138683] [GitHub Education]
### Hi everyone,

Hello GitHub Education team,

I'm experiencing some difficulties with my Student Developer Pack application, specifically regarding the upload of my certificate of enrollment. I have a few questions and would greatly appreciate your guidance.

1. **Document Upload**: I'm unable to find a way to attach my certificate of enrollment. The only option I see is to use a webcam to capture the document. Is there an alternative method to upload this document directly?

2. **Location Verification**: Currently, I'm away from my university area due to the Korean Chuseok holiday. I'll be returning to my campus vicinity next week. Should I wait until I'm back near my school to resubmit my application for proper location verification?

Thank you in advance for your help and understanding. I'm looking forward to resolving these issues and completing my application successfully.

Best regards,
Seungheon Lee

**Comment by davevad93** [comment]
Hi @Githarold ,

Last year there was an "Upload" button to submit the academic proof in `PNG` or `JPG` format, but now it seems that only the camera option is available, so you should take a picture with your PC webcam or smartphone camera.

As for the location, I suggest you to wait until you are back near your school, otherwise your application will be rejected.

For more information, check the docs on [applying to GitHub Education as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) and also the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352).
